index.js

```diff
@@ -1,6 +1,9 @@
-var methods = require('methods');
+var methods = require('methods'),
+    extend = require('extend');
```

Comment:
This is uncommon indenting because there's 4 spaces instead of 2, which almost any JS formatter will bring back to 1 which makes it like this:

```js
var methods = require('methods'),
  extend = require('extend');
```


So please use 1 of the following;

```js
var methods = require('methods');
var extend = require('extend');
```

Or 

```js
var methods = require('methods')
  , extend = require('extend');
```

_And yes this might seem OCD driven_